source/whats-new.txt

```diff
@@ -33,6 +33,15 @@ Learn what's new in:
 What's New in 1.11
 --------------------
 
+.. important:: Upgrade to Version 1.11.2 or Higher 
+
+  The 1.11.2 Go driver contains the following bug fixes:
+  
+  - ``bson.Raw.String`` outputs the correct Extended JSON type for timestamps when 
+    the BSON document contains a ``UTC Timestamp`` field.
```

Comment:
Issue: I'm not sure that UTC Timestamp is the actual type name, could it be just [date](https://www.mongodb.com/docs/manual/reference/bson-types/#date)?

Suggestion: UTC Timestamp not monospaced

Reply:
Makes sense. Looks like [the BSON type is "Timestamp"](https://github.com/mongodb/specifications/blob/master/source/extended-json.rst#conversion-table) but removing monospace seems like the safest option.